submitted_models/marble_hd2_sensor_config_3/launch/spawner.rb

```diff
@@ -0,0 +1,80 @@
+def spawner(_name, _modelURI, _worldName, _x, _y, _z, _roll, _pitch, _yaw)
+  <<-HEREDOC
+  <spawn name='#{_name}'>
+    <name>#{_name}</name>
+    <allow_renaming>false</allow_renaming>
+    <pose>#{_x} #{_y} #{_z + 0.2} #{_roll} #{_pitch} #{_yaw}</pose>
+    <world>#{_worldName}</world>
+    <is_performer>true</is_performer>
+    <sdf version='1.6'>
+    <include>
+      <name>#{_name}</name>
+      <uri>#{_modelURI}</uri>
+      <!-- Diff drive -->
+      <plugin filename="libignition-gazebo-diff-drive-system.so"
+              name="ignition::gazebo::systems::DiffDrive">
+        <left_joint>front_left_wheel_joint</left_joint>
+        <left_joint>front_middle_left_wheel_joint</left_joint>
+        <left_joint>rear_middle_left_wheel_joint</left_joint>
+        <left_joint>rear_left_wheel_joint</left_joint>
+        <right_joint>front_right_wheel_joint</right_joint>
+        <right_joint>front_middle_right_wheel_joint</right_joint>
+        <right_joint>rear_middle_right_wheel_joint</right_joint>
+        <right_joint>rear_right_wheel_joint</right_joint>
+        <wheel_separation>#{0.525}</wheel_separation>
+        <wheel_radius>0.129</wheel_radius>
+        <topic>/model/#{_name}/cmd_vel_relay</topic>
+        <min_velocity>-1</min_velocity>
+        <max_velocity>1</max_velocity>
```

Comment:
These velocities seem pretty low - are these the correct values, or should they be higher?

Reply:
Owing to COVID-19 we do not have validation data. I will ask MARBLE if that has changed in the past couple weeks. Those were the values assigned in leu of not having validation data.

Reply:
Owing to COVID-19 we do not have validation data. I will ask MARBLE if that has changed in the past couple weeks. Those were the values assigned in leu of not having validation data.

Reply:
I know that several other teams/model submissions have faced the same validation data issues due to COVID-19. If you'd like, you can increase the magnitude of these velocities up to `4m/s` until validation data is received - just pick whatever value you believe is reasonable.

Reply:
The HD2 model will continue to have 1 m/s maximum velocity (matching X1 and Husky) until validation can be provided. Matching the speed to the most similar pre-existing model avoids any new model having a speed advantage that has not been validated.